front/src/index.html

```diff
@@ -55,81 +84,42 @@
                     loggedUser: null,
                     roomEditorHidden: true,
                     currentTypedMessage: "",
-                    rooms: [
-                        {
-                            "_id": {
-                                "$oid": "6071ac5176e5bd456a859d9d"
-                            },
-                            "description": "ma description",
-                            "title": "Room 1",
-                            "user": []
-                        },
-                        {
-                            "_id": {
-                                "$oid": "6071ac5176e5bd456a859d9f"
-                            },
-                            "description": "ma description",
-                            "title": "Room 2",
-                            "user": []
-                        },
-                        {
-                            "_id": {
-                                "$oid": "6071ac5176e5bd456a859d94"
-                            },
-                            "description": "ma description",
-                            "title": "Room 3",
-                            "user": []
-                        }
-                    ],
-                    users: [
-                        {
-                            "_id": {
-                                "$oid": "607ae876f59d21d5a50ac3c9"
-                            },
-                            "pseudo": "mon pseudo1",
-                            "roomsId": []
-                        },
-                        {
-                            "_id": {
-                                "$oid": "607ae876f59d21d5a50ac3c2"
-                            },
-                            "pseudo": "mon pseudo2",
-                            "roomsId": []
-                        }
-                    ],
-                    messages: [
-                        {_id : {$oid: "459I548383"},content: "Bonjour tata", author: "toto",  timestamp: 1618587750035},
-                        {_id : {$oid: "4459493924"},content: "Salut toto", author: "tata",  timestamp: 1618587750035 },
-                    ]
+                    rooms: [],
+                    users: [],
+                    messages: []
                 }),
+                mounted() {
+                    this.fetchRooms();
+                },
                 methods: {
+                    fetchRooms() {
+                        if(this.loggedUser === null) return;
+                        console.log(this.loggedUser);
+                           debugger;
+                            srv.getUserByid(this.loggedUser.id).then(u=> {
+                                this.loggedUser = u;
+                                this.rooms = u.rooms;
+                                console.log(this.rooms);                             
+                            });
+                            this.rooms = this.loggedUser.rooms;
+                    },
```

Comment:
this needs to go in scratchy service,
it would be a `getRoomsOfUser(User) => Promise<Room[]>`